streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/controller/ApplicationBuildPipelineController.java

```diff
@@ -49,13 +49,9 @@ public class ApplicationBuildPipelineController {
   @PermissionScope(app = "#appId")
   @PostMapping(value = "build")
   @RequiresPermissions("app:create")
-  public RestResponse buildApplication(Long appId, boolean forceBuild) {
-    try {
-      boolean actionResult = appBuildPipeService.buildApplication(appId, forceBuild);
-      return RestResponse.success(actionResult);
-    } catch (Exception e) {
-      return RestResponse.success(false).message(e.getMessage());
```

Comment:
The FE will logical processing based on the controller response. If an exception is thrown here, it will cause a bug in the front end.